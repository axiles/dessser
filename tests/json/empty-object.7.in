{ "bar": [ {}, {}, null, { "glop":false } , {"baz":[-1, -2, null]} ] }
