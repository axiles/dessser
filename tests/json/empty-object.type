{
  foo: string?;
  bar: {
    baz: i32?[]?;
    glop: bool?;
  }?[]?;
}
