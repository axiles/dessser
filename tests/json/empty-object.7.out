(null ((null null) (null null) null (null F) ((-1 -2 null) null)))
